packages/core/src/components/content/content.ios.vars.scss

```diff
@@ -7,7 +7,4 @@
 $content-ios-font-family:               $font-family-ios-base !default;
 
 /// @prop - Background color of the outer content
-$content-ios-outer-background:          #efeff4 !default;
-
-/// @prop - Background color of the content when making transition
-$content-ios-transition-background:     #000 !default;
```

Comment:
I couldn't find anywhere this way used?

Reply:
Ah okay so `$content-ios-outer-background` is added by a class here: https://github.com/ionic-team/ionic/blob/core/packages/core/src/components/content/content.ios.scss#L14-L16

I've always thought we needed to rethink that class, but it is used in a lot of places.

And `$content-ios-transition-background` isn't implemented but it should be used by the ios transition here: https://github.com/ionic-team/ionic/blob/master/src/components/content/content.ios.scss#L40

It may be implemented another way though. Also, I'm not sure that this is a variable that needs theming.